README.md

```diff
@@ -1889,3 +1889,5 @@ API | Description | Auth | HTTPS | CORS |
 
 ## License
 [MIT](LICENSE) (c) 2022 public-apis
+
+| [NASA](https://api.nasa.gov) | NASA data, including imagery | No | Yes | Yes |
```

Comment:
Question: should this be higher up in one of the previous categories?

This seems to be appened to the end of the file under the License and would more discoverable placed conventionally in one of the other categories above, maybe in [Science & Math](https://github.com/public-apis/public-apis/blob/master/README.md#science--math) ?